src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+function getMarriageStatus(person) {
+  return person.sex === 'm'
+    ? `${person.partnerName} is my wife`
+    : `${person.partnerName} is my husband`;
+}
+
+export const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is `}
+        {person.name}
+      </h2>
+
+      {person.age && (
+        <p className="Person__age">
+          {`I am `}
+          {person.age}
```

Comment:
```suggestion
          I am {age}
```

Reply:
Linter won't let me do it in a single row 😢